libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/pwmout_api.c

```diff
@@ -74,40 +125,170 @@ static __IO uint16_t *PWM_MATCH[] = {
     &PWMPWBFR_2E,    // PWM13_PIN
 };
 
+static __IO uint16_t *MTU2_PWM_MATCH[MTU2_PWM_NUM][MTU2_PWM_SIGNAL] = {
+    { &MTU2TGRA_2, &MTU2TGRB_2 },       // MTU2_PWM0_PIN
```

Comment:
Hello

Thanks your comments.
I would like to declare with volatile. "__IO" Means the volatile declaration.
So, I would like to leave the status quo.

How about our proposal ?